client/src/pages/Blog.jsx

```
@@ -1,10 +1,133 @@
+          {data.title}
+        </h1>
+        <h2 className='my-5 max-w-lg truncate mx-auto'>{data.subtitle}</h2>
+        <p className='inline-block py-1 px-4 rounded-full mb-6 border text-sm border-primary/35 bg-primary/5 font-medium text-primary'>
+          Micheal Brown
+        </p>
+      </div>
+      <div className='mx-5 max-w-5xl md:mx-auto my-10 mt-6'>
+        <img src={data.image} alt='' className='rounded-3xl mb-5' />
+        <div
+          className='rich-text max-w-3xl mx-auto'
+          dangerouslySetInnerHTML={{ __html: data.description }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)